migrations/V174__Add_Solution_Architect_Role.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TYPE TEAM_ROLE ADD VALUE 'SOLUTION_ARCHITECT';
+ALTER TYPE DISCUSSION_USER_ROLE RENAME VALUE 'IT_ARCHITECT' TO 'SOLUTION_ARCHITECT';
```

Comment:
Per Steven's point, I'm not sure if this is meant to be renaming `IT_ARCHITECT` to `SOLUTION_ARCHITECT`, but perhaps just introducing a new `SOLUTION_ARCHITECT` value